account_cutoff_base/README.rst

```diff
@@ -7,15 +7,15 @@ Account Cut-off Base
    !! This file is generated by oca-gen-addon-readme !!
    !! changes will be overwritten.                   !!
    !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-   !! source digest: sha256:067c3a39abe7ad82a6363f9758669477eb84956bc7f6d793591d9501a39a1c85
+   !! source digest: sha256:8594085b2ee485fa468f0048cf1a7c979b49a57ff8c96c5c60960d949c73c3d7
    !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
 .. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
     :target: https://odoo-community.org/page/development-status
     :alt: Beta
-.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
-    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
-    :alt: License: AGPL-3
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
```

Comment:
This commit for migrating the module has included a licence change from AGPL to LGPL. We can't do a silent licence change like this ! It needs to be discussed publically !